ehrql/file_formats/main.py

```diff
@@ -23,6 +24,9 @@
 
 
 def write_rows(filename, rows, column_specs):
+    if filename is None:
```

Comment:
There's a check for `if filename is not None` on row 38 which should never be False now, I think

Reply:
Ah, removed in next commit, nvm